### PR TITLE
Updated two tests to use utils.getPackageDir

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

**Comment by jhoblitt** [comment]
@r-owen any idea as to what the correct way to fall back for the getEupsProductName method?  The only caller github search can find is in pipe_base.

https://github.com/lsst/daf_butlerUtils/blob/master/python/lsst/daf/butlerUtils/cameraMapper.py#L500-L514

https://github.com/lsst/pipe_base/blob/767c8ebabff58ced76777cb21c0cd267a6e52b6f/python/lsst/pipe/base/argumentParser.py#L355

**Comment by r-owen** [comment]
What do you mean by "fall back"? If you mean "handle the case that the package is not setup but you want to proceed anyway" then I suggest enclosing in a try/except Exception loop.

**Comment by jhoblitt** [comment]
What I meant was, how do you find the package name without the presence of eups?

**Comment by r-owen** [comment]
We won't be able to answer that question until we have a packaging system that is different than eups. At that point the utils function will have to be updated. There is one exception: if the desired path is within the same package (e.g. a unit test in package "foo" wants to find something else in the package "foo") then it can use form a path based on the global variable "__file__" in the test itself.

**Comment by RobertLuptonTheGood** [comment]
Well, if you know that eups uses $FOO_DIR for the root directory you can write an implementation that hides the fact that we're using environment variables set by eups -- I thought that that was what you'd already done?

**Comment by jhoblitt** [comment]
@RobertLuptonTheGood The method @r-owen added answers the question, "what is the product directory for package X?".  The question in getEupsProductName(), is "what is my package name?".

**Comment by jhoblitt** [comment]
@ktlim How do you feel about turning CameraMapper.getEupsProductName() in a stub method that subclasses are required to implement?  That seems to be what subclasses, at least for code that's on github, are doing already.

https://github.com/search?utf8=%E2%9C%93&q=user%3Alsst+getEupsProductName&type=Code&ref=searchresults

**Comment by ktlim** [comment]
Only a couple of subclasses are short-cutting the auto-discovery process; there are several more in other `obs_*` packages that are not., and you'd have to make sure you found all of them if you made this change.  I think that the `CameraMapper.getEupsProductName()` method and its usage downstream in `pipe_base` to find configuration override files should be replaced by something else that provides access to camera-specific data.  It's not entirely clear to me right now whether this should always be the currently-installed version of the data (what is currently done) or a bitemporal lookup of the data.  But that's for further down the road, and this change seems like a small one that factors things better and makes things less complex, at the cost of a bit more boilerplate for `obs_*` package implementers.

Bottom line: renaming this to `getPackageName()` and making it a required override is fine.

**Comment by jhoblitt** [comment]
I've opened a separate issue for the `getEupsProductName` rename and conversion. https://jira.lsstcorp.org/browse/DM-2789

File: tests/ticket1580.py [hunk 1]
```diff
@@ -29,11 +29,18 @@
...
+    print "afwdata not setup; not running test."
+    sys.exit(0)
+
+try:
+    from lsst.obs.lsstSim import LsstSimMapper
+except ImportError:
+    print "obs_lsstSim not setup; not running test."
     sys.exit(0)
 
 import lsst.daf.persistence as dafPersist
```
Comment (timj) [inline tests/ticket1580.py]: I think you need to remove the old lsst.obs.lsstSim import.
